### DISCUSSION TITLE
Rollback of localized field to non-localized field

### DISCUSSION THREAD

**Discussion by agdrs** [pimcore/pimcore#17193] [General]
Hi,

we checked our database and realized, that fields, which were defined as localized field and afterwards changed to a non-localized field are still present in the localized_tables.
In my opinion there should exist some kind of rollback that these fields are deleted.
Or am i missing the point here, that it is an expected behaviour?
What do you think?

Thanks and regards
Andy

**Comment by kingjia90** [comment]
Hi Andy,

Thank you for your input!

I think it is a little "unconventional" use case and would be technically hard to guarantee that it would be working that seamlessly to be officially supporting it, but you can always try it and open a PR for it, i'll be glad to have a look on it.

Best regards,
JJ
